hepcrawl/extractors/jats.py

```diff
@@ -68,6 +68,16 @@ def _get_keywords(self, node):
                     free_keywords.append(keyword)
         return free_keywords, classification_numbers
 
+    def _get_public_notes(self, node):
+        """Return list of public notes from node."""
+        public_notes = []
+        for note in node.xpath('//notes[@notes-type]'):
+            for keyword in note.xpath('p/text()').extract():
+                public_notes.append({
+                    'value': keyword,
+                })
+        return public_notes
+
```

Comment:
This Jats module is used also by the edp spitder, can you add some tests on the wsp side at least, so when we go back here when `polishing` the edp spider, we don't break the wsp one?

Hmmmpf, I'll add a task (high prio) to add tests for wsp at the same level as arxiv, using the pipeline instead of just the spider to make sure that anything outputted by the spider ends up being schema compatible.

Because, on `_get_public_notes` for example, the source is missing in the public note:
    https://github.com/inspirehep/inspire-schemas/blob/d45edc53af7dd996d281e9b245b6eff32b4d19ed/inspire_schemas/records/elements/sourced_value.yml